query-planner-js/src/__tests__/buildPlan.test.ts

```diff
@@ -4285,3 +4370,55 @@ describe('__typename handling', () => {
     `);
   });
 });
+
+describe('mutations', () => {
```

Comment:
Note to reviewer: that test is not really directly related to this patch, but I notice that `buildPlan.test.ts` had no simple Mutation test. Mutation were still tested elsewhere (in the plan execution tests mostly, but also in plenty of older test files) but as I often use that one test file as a shortcut for "have I broken query planning", it's convenient to have at least a simple mutation test there.